src/history/hs_rec.c

```diff
@@ -684,7 +684,6 @@ __wt_hs_insert_updates(
     }
 
     WT_ERR(__wt_block_manager_named_size(session, WT_HS_FILE, &hs_size));
```

Comment:
Then why we need call this anymore?

Reply:
@quchenhao There is an if statement below which is using the variable hs_size. Do you think we still need that if statement?

Reply:
We need to figure out what is file_max and if is still relevant.

Reply:
It looks like the code is trying to make sure that the size of the HS file does not exceed the maximum we have decided for this type of table. I think it may make more sense if we move this to where we are updating the statistic.